docs/pools/pools-processes/lending.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Pool Lending Process 
+sidebar_position: 1
```

Comment:
@nabaelie  sidebar_position should be unique for each file , consider changing to 

`sidebar_position: 2`